src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -243,13 +243,27 @@ Func<object>	DefenseStep()
 	
 	var RatTargetNext = SetRatTarget?.OrderBy(target => target?.DistanceMax ?? int.MaxValue)?.FirstOrDefault();
 	
+	//Populating array to check if drones are fighting or not
+	Sanderling.Interface.MemoryStruct.IUIElementText[] droneArray = Sanderling.MemoryMeasurementParsed?.Value?.WindowDroneView?.FirstOrDefault()?.LabelText.ToArray();
```

Comment:
If I understand correctly, this variable will contain all labels of the drones window, therefore naming this `droneArray` seems misleading

Why use `LabelText` here?
Would it not be sufficient to instead read the drones entries under `WindowDroneView?.ListView?.Entry`?

Do you see the entries for the drones under `WindowDroneView?.ListView?.Entry` in the API Explorer?